nuxeo-ai-model/src/main/java/org/nuxeo/ai/bulk/ExportInitComputation.java

```diff
@@ -278,19 +277,10 @@ public DatasetExport createDataset(CoreSession session, String query, Set<Proper
         DatasetExport adapter = doc.getAdapter(DatasetExport.class);
         adapter.setQuery(query);
         adapter.setSplit(split);
-        List<DatasetExport.IOParam> inputForAdp = inputs.stream().map(p -> new DatasetExport.IOParam() {
-            {
-                put("name", p.getName());
-                put("type", p.getType());
-            }
-        }).collect(Collectors.toList());
+
+        List<Map<String, String>> inputForAdp = inputs.stream().map(p -> Map.of("name", p.getName(), "type", p.getType())).collect(Collectors.toList());
         adapter.setInputs(inputForAdp);
-        List<DatasetExport.IOParam> outputForAdp = outputs.stream().map(p -> new DatasetExport.IOParam() {
-            {
-                put("name", p.getName());
-                put("type", p.getType());
-            }
-        }).collect(Collectors.toList());
```

Comment:
avoiding double instantiation. Map.of is not available for 10.10 but there I will have deal with it differently